lib/core/backup_service.dart

```diff
@@ -206,6 +230,16 @@ class BackupService {
         json.decode(transactionDescriptionFile.readAsStringSync()) as Map<String, dynamic>;
     final descriptionsMap = jsonData.map((key, value) =>
         MapEntry(key, TransactionDescription.fromJson(value as Map<String, dynamic>)));
+
+    if (!_transactionDescriptionBox.isOpen) {
+      final transactionDescriptionsBoxKey = await getEncryptionKey(secureStorage: secureStorageShared, forKey: TransactionDescription.boxKey);
+      final transactionDescriptionBox = await CakeHive.openBox<TransactionDescription>(
+        TransactionDescription.boxName,
+        encryptionKey: transactionDescriptionsBoxKey,
+      );
+      await transactionDescriptionBox.putAll(descriptionsMap);
+      return;
+    }
```

Comment:
why would it be closed? it gets opened in main before the app starts
and if indeed it's necessary, please use `this.secureStorage` instead

Reply:
ok so this function `_reloadHiveWalletInfoBox` is the one that closes it, but it also closes all boxes and just open the walletInfo box, so let's make sure that's the expected behavior, cuz I don't think it is

Reply:
It is an issue that [Ramo reported](https://cakewallet.slack.com/archives/C04CSQESA58/p1728607357598899?thread_ts=1728577025.568509&cid=C04CSQESA58), and I was able to reproduce later, with this fix there is no error